index.html

```diff
@@ -101,19 +101,19 @@ <h5>I am a Software Developer</h5>
         <div class="mobile-version-icons text-white font-size-medium">
           <ul class="list-none show-on-small hide-on-large">
             <li>
-              <a href="#"><i class="fab fa-github icons"></i></a>
+              <a href="#" aria-label="angelist icon"><i class="fab fa-github icons"></i></a>
             </li>
             <li>
-              <a href="#"><i class="fab fa-linkedin-in"></i></a>
+              <a href="#" aria-label="angelist icon"><i class="fab fa-linkedin-in"></i></a>
             </li>
             <li>
-              <a href="#"><i class="fab fa-angellist"></i></a>
+              <a href="#" aria-label="angelist icon"><i class="fab fa-angellist"></i></a>
             </li>
             <li>
-              <a href="#"><i class="fab fa-twitter"></i></a>
+              <a href="#" aria-label="angelist icon"><i class="fab fa-twitter"></i></a>
             </li>
             <li>
-              <a href="#"><i class="fab fa-medium-m"></i></a>
+              <a href="#" aria-label="angelist icon"><i class="fab fa-medium-m"></i></a>
```

Comment:
[OPTIONAL]

- [ ] Kindly change the aria-labels for each

Reply:
Okay, thanks for the review, I will make the changes.